src/server/playwright.ts

```diff
@@ -35,6 +36,7 @@ export class Playwright extends SdkObject {
   readonly firefox: Firefox;
   readonly webkit: WebKit;
   readonly options: PlaywrightOptions;
+  _portForwardingServer: PortForwardingServer | undefined;
```

Comment:
We should either make this private, or leave it public and remove all the methods below. Otherwise, it is not clear who owns this property.